packages/common/src/utils/stream-utils.ts

```diff
@@ -142,8 +150,8 @@ export class StreamUtils {
     if (state.metadata?.model) {
       cloned.metadata.model = state.metadata.model.toString()
     }
-    if (state.next?.metadata?.model) {
```

Comment:
this is safe to delete because only TileDocument does the thing with using `next` for unanchored changes to content and metadata.  Model and ModelInstanceDocument streams never have a `next` state, and those are the only ones that use `model` in their metadata

Reply:
Makes sense that we never have `model` in next.